README.md

```diff
@@ -190,6 +190,18 @@ toLowerCase({ nested: { prop: { 1: { msg: 'Hello' }, 2: { msg: 'Hi' }, 3: { msg
 
 When applying operations on a path immutad●t tries to create the minimum of objects or arrays needed to guarantee your data structure to be immutable.
 
+[mweststrate](https://github.com/mweststrate) made a [simple benchmark](https://github.com/Zenika/immutadot/tree/master/packages/immutadot-benchmark/src/performance.spec.js) on the performance of [immer](https://github.com/mweststrate/immer) and other methods for setting a property on multiple items of a todo list and keep it immutable. Here are the results with immutad●t in addition:
```

Comment:
I think we should mention here some information on hardware (MacBook Pro...) and software (node 8, without parallelism)...